src/main/java/uk/gov/hmcts/reform/sscs/utility/CreatePanelJsonFromCsv.java

```diff
@@ -0,0 +1,114 @@
+package uk.gov.hmcts.reform.sscs.utility;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class CreatePanelJsonFromCsv {
+
+    String delimiter = ",";
+    String lineData;
+    //    csv file should have five columns: ccd_key, Session Category 1, Other Session Category, Panel Requirements 1, Panel Requirements Other
+    //    do a replace all for ',' and '& ' and replace with empty string (command + shift + H)
+    //    replace any empty cells with two apostrophes ('')
+    //    use online csv converter to convert xls to csv
+    File csvFile = new File("src/main/resources/reference-data/johTierAutomateAttempt.csv");
+    static int benefitIssueCodeIndex = 0;
+    static int category1Index = 1;
+    static int category2Index = 2;
+    static int panel1ColumnIndex = 3;
+    static int panel2ColumnIndex = 4;
+
+    public void convertCsvToJson() {
+        try (BufferedReader read = new BufferedReader(new FileReader(csvFile))) {
+            JSONArray panelCompositionObject = new JSONArray();
+
+            while ((lineData = read.readLine()) != null) {
```

Comment:
is it not possible to follow the approach used in MigrationDataEncoderApp where we read the whole csv in one go into map/list, and then inspect it through java